Engine/src/Engine/Audio.h

```diff
@@ -0,0 +1,86 @@
+#pragma once
+#ifndef _AUDIO_H_
+#define _AUDIO_H_
+
+#include <iostream>
+#include "..\irrKlang\irrKlang.h"
```

Comment:
shouldn't use relative paths to include irrKlang. This should already be available from CMake as an additional include